deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/Convolution1DLayer.java

```diff
@@ -178,10 +179,30 @@ public Builder stride(int stride) {
          * @param padding Padding value
          */
         public Builder padding(int padding) {
-            this.padding[0] = padding;
+            this.setPadding(new int[]{padding});
             return this;
         }
 
+        @Override
+        public void setKernelSize(int[] kernelSize) {
```

Comment:
Let's make all of these ```setX(int[] x)``` varargs methods - that allows users (and indeed the builder methods) to do things like ```setKernelSize(kernelSize);``` instead of ```setKernelSize(new int[]{kernelSize});```
That should still be an override. This comment applies to other layers too, not just this one.